### PR TITLE
python311Packages.stdlibs: 2022.10.9 -> 2023.11.2

### PR DESCRIPTION
**Pull request by fabaff**
Diff: https://github.com/omnilib/stdlibs/compare/refs/tags/v2022.10.9...v2023.11.2

Changelog: https://github.com/omnilib/stdlibs/blob/2023.11.2/CHANGELOG.md

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/stdlibs/default.nix [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 buildPythonPackage rec {
   pname = "stdlibs";
-  version = "2022.10.9";
+  version = "2023.11.2";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
```
Comment (natsukium) [inline pkgs/development/python-modules/stdlibs/default.nix:13]: https://github.com/omnilib/stdlibs/blob/a2f0bbf6198589218abe8d38878acafc2e89af21/pyproject.toml#L20
```suggestion
  disabled = pythonOlder "3.8";
```

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 265325` at 2e2e672d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built successfully:</summary>
  <ul>
    <li>python310Packages.stdlibs</li>
    <li>python310Packages.ufmt</li>
    <li>python310Packages.usort</li>
    <li>python311Packages.stdlibs</li>
    <li>ufmt (python311Packages.ufmt)</li>
    <li>usort (python311Packages.usort)</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 265325` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>python310Packages.stdlibs</li>
    <li>python310Packages.stdlibs.dist</li>
    <li>python310Packages.ufmt</li>
    <li>python310Packages.ufmt.dist</li>
    <li>python310Packages.usort</li>
    <li>python310Packages.usort.dist</li>
    <li>python311Packages.stdlibs</li>
    <li>python311Packages.stdlibs.dist</li>
    <li>ufmt (python311Packages.ufmt)</li>
    <li>ufmt.dist (python311Packages.ufmt.dist)</li>
    <li>usort (python311Packages.usort)</li>
    <li>usort.dist (python311Packages.usort.dist)</li>
  </ul>
</details>

File: pkgs/development/python-modules/stdlibs/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/stdlibs/default.nix]: Thanks for checking.

**Pull request merged**